app/models/course/assessment/programming_evaluation.rb

```diff
@@ -0,0 +1,62 @@
+# Represents one pending programming question evaluation.
+#
+# This is different from a job in that jobs are assigned to an arbitrary worker. These
+# programming question evaluations can only be run on a specific worker,
+#  a) has permission to access the course
+#  b) has the programming language runtime to evaluate this submission
+#
+# Like trackable jobs, elements in this table are ephemeral.
+class Course::Assessment::ProgrammingEvaluation < ActiveRecord::Base
```

Comment:
I can't make it run on the evaluator, the evaluator will query Coursemology for outstanding jobs. The idea is that evaluators will be completely separate from our web application and query the web application for things to evaluate. That's why there's an assignment of an evaluation to a user, because the evaluators need to log in also